src/plugins/registry.js

```diff
@@ -74,8 +74,24 @@ module.exports = function (bookshelf) {
   // can't include it with the rest of the relational methods.
   const morphTo = Model.prototype.morphTo;
   Model.prototype.morphTo = function(relationName) {
```

Comment:
I don't really like that the API gets slightly changed. But I don't have any better idea to solve it.